webapp/src/components/CreateMeetingNoteForm.tsx

```diff
@@ -0,0 +1,114 @@
+import { Alert, Snackbar, Stack, TextField } from '@mui/material';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import { LoadingButton } from '@mui/lab';
+import React, { FormEventHandler, useState } from 'react';
+
+import { EntityId } from '../types/api';
+interface CreateMeetingNoteFormProps {
+  agendaOwningParticipantId: EntityId;
+  meetingId: EntityId;
+  onMeetingNoteChanged?: (id: EntityId) => void;
+  sortOrder: number | undefined;
```

Comment:
Optional props are specified using `?`, not `| undefined`. However, it looks like `sortOrder` is required to be a number based on the validation in the api, so this shouldn't be optional.

Reply:
Thanks.  I removed the `| undefined`.  `sortOrder` is initialized to `1` or set to another value depending on the number of notes in an agenda.